pyerrors/correlators.py

```diff
@@ -701,7 +698,7 @@ def second_deriv(self, variant="symmetric"):
         else:
             raise ValueError("Unknown variant.")
 
-    def m_eff(self, variant='log', guess=1.0):
+    def m_eff(self, variant: str='log', guess: float=1.0) -> "Corr":
```

Comment:
A very naive question and probably that is just something that was automatically done: When shoult it be `Optional[str]` (as above) and when `str` (as here)?

Reply:
I think it should be `Optional[str]` when the value can also be `None`, I think in this case one really needs to pass a string, right?

Reply:
Optional[X] is equivalent to X | None (or Union[X, None]).

https://docs.python.org/3/library/typing.html#typing.Optional